core/src/main/scala/org/apache/spark/deploy/mesos/ui/DriverPage.scala

```diff
@@ -50,7 +50,7 @@ private[ui] class DriverPage(parent: MesosClusterUI) extends WebUIPage("driver")
     val driverDescription = Iterable.apply(driverState.description)
     val submissionState = Iterable.apply(driverState.submissionState)
     val command = Iterable.apply(driverState.description.command)
-    val schedulerProperties = Iterable.apply(driverState.description.schedulerProperties)
+    val schedulerProperties = Iterable.apply(driverState.description.sc.getAll.toMap)
```

Comment:
One thing that concerns me besides this being a bit verbose is that it does not provide whose properties these are. 
It would be ok i guess if sc properties are propagated without any modification to all involved components, is this the case?